quint/src/reflection.proto

```diff
@@ -0,0 +1,136 @@
+// Copyright 2016 gRPC authors.
```

Comment:
Towards my other point re: needing a proto file: this proto file is more complex than the Apalache proto file. So we’d win on reduced code here too.

Reply:
My typescript is rusty. The idea is to use a gRPC client that supports the reflection API. I can't find it on npm. So this is my solution. You can definitely statically compile reflection.proto to typescript and import it in the Quint package.

Reply:
but if we are gonna statically compile, we can just statically compile the apalache proto file :D